### PR TITLE
docker : dockerfile edited for multi-dev environments

### PR DESCRIPTION
**Pull request by Bog0sso**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 WORKDIR /django
-COPY requirements.txt requirements.txt
-RUN pip install -r requirements.txt
-
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
 COPY . .
+EXPOSE 8000
+RUN python manage.py migrate
+CMD [ "python", "manage.py","runserver"]
```
Comment (Linzo99) [inline Dockerfile]: this command will server django on localhost which will make it impossible for you to access from your computer.

Use this instead
CMD ["python", "manage.py", "runserver", "0.0.0.0:8000"]

Comment (Bog0sso) [reply Dockerfile]: I have updated the code. Everything is fine now.

**Comment by Kepsondiaz** [comment]
hello @Bog0sso can you add the command to run it 
` cd xassida-api `
` docker build -t dockerfile  . `

**Comment by Linzo99** [comment]
> hello @Bog0sso can you add the command to run it `cd xassida-api` `docker build -t dockerfile .`

Yeh we can add it in the README

**Comment by Linzo99** [comment]
Good, il reste juste un script pour remplir la base de donnée,  mais ça peut attendre.

**Review by Linzo99 — APPROVED** [review]

**Pull request merged**